security/pfSense-pkg-suricata/files/var/db/suricata/sidmods/modifysid-sample.conf

```diff
@@ -19,5 +19,13 @@
 # "HTTP_PORTS" "HTTPS_PORTS"
 
 # multiple sids can be specified as noted below:
-# 302,429,1821 "\$EXTERNAL_NET" "\$HOME_NET"
+# 302,429,1821 "$EXTERNAL_NET" "$HOME_NET"
```

Comment:
The code is escaping it itself. See line 2506. Actually that way you can't use regex for replacement. pulledpork supports replacement by regex in such case you need to escape it. I thought of removing the preg_quote so that the current example would be correct. But dropped that idea to no break the config for people who have noticed that escaping is not needed as displayed in the example.